FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -1234,12 +1249,15 @@ extension Auth: AuthInterop {
         requestConfiguration: self.requestConfiguration
       )
       #if os(iOS)
-        self.wrapInjectRecaptcha(request: request,
-                                 action: AuthRecaptchaAction.getOobCode) { result, error in
-          if let completion {
-            DispatchQueue.main.async {
-              completion(error)
-            }
```

Comment:
Since `error` here is optional, important that new diff calls error in both success and failure.